packages/create-block/docs/external-template.md

```diff
@@ -74,6 +74,7 @@ The following configurable variables are used with the template files. Template
 -   `customScripts` (default: {}) – the list of custom scripts to add to `package.json` . It also allows overriding default scripts.
 -   `npmDependencies` (default: `[]`) – the list of remote npm packages to be installed in the project with [`npm install`](https://docs.npmjs.com/cli/v8/commands/npm-install) when `wpScripts` is enabled.
 -   `npmDevDependencies` (default: `[]`) – the list of remote npm packages to be installed in the project with [`npm install --save-dev`](https://docs.npmjs.com/cli/v8/commands/npm-install) when `wpScripts` is enabled.
+-   `customPackageJSON` (no default) - allows definition of additional properties for the generated package.json file.
```

Comment:
We have a bit of overlap with `customScripts`, `npmDependencies`, and `npmDevDependencies`. It doesn’t have to be a bad thing as we would have to put special care to handle them in a similar way through `customPackageJSON`.

Reply:
That's a good point. I wonder if we just filter those keys out of `customPackageJSON`?

Reply:
It isn’t a blocker. We can look into it once it starts to confuse people.